.circleci/config.yml

```diff
@@ -453,6 +438,38 @@ commands:
             fi
         #
 
+  halt-build-archive:
```

Comment:
New command: Halt build if existing build archive is already stored in Google bucket. Shell script are copied from command `conditionally-launch-build-and-store`.